src/experiment_prototype/scan_classes/averaging_periods.py

```diff
@@ -10,11 +10,23 @@
     :copyright: 2018 SuperDARN Canada
     :author: Marci Detwiller
 """
+# built-in
+import inspect
+from pathlib import Path
 
+# third party
```

Comment:
```suggestion
```
I like the structure of splitting imports into three separate categories. These repeated comments aren't really needed though, IMO.